docs/api.md

```diff
@@ -131,6 +131,50 @@ A Keras model instance.
 
 **References**
 
+- [Back to Simplicity:
+  How to Train Accurate BNNs from Scratch?](https://arxiv.org/abs/1906.08637)
+
+## DenseNet28<a class="headerlink code-link" style="float:right;" href="https://github.com/larq/zoo/blob/master/larq_zoo/densenet.py#L119" title="Source code"></a>
+
+```python
+DenseNet28(include_top=True,
```

Comment:
To avoid ambiguity when comparing full precision architectures should we call this BinaryDensenet? Of course we could `import Densenet as BinaryDensenet` but being explicit (and calling it exactly as in the paper) might prevent some confusion.

Reply:
I agree. That seams like a good idea.